src/main/scala/scalaz/parsers/core/TransformClass.scala

```diff
@@ -44,6 +47,8 @@ trait TransformClass[=>:[_, _]] extends StrongClass[=>:] with CategoryClass[=>:]
 
 object Transform {
 
+  import types._
```

Comment:
This is what I'd like to avoid. From user perspective, requirement to use additional imports is a big downside. Remember, imports are not discoverable.
It doesn't make much difference how many imports are there at our current stage, but since you intend to improve upon code organization, I'd say it's expected for code to be removed rather than added 😉

Reply:
I do tend to like making things explicit but thinking about it some more, I agree this is probably taking it too far.